python/sklearnserver/README.md

```diff
@@ -104,3 +104,69 @@ To run static type checks:
 mypy --ignore-missing-imports sklearnserver
 ```
 An empty result will indicate success.
+
+## Building your own XGBoost Server Docker Image
+
+You can build and publish your own image for development needs. Please ensure that you modify the kfservice files for XGBoost in the api directory to point to your own image.
+
+To build your own image, run
+
+```bash
+docker build -t animeshsingh/sklearnserver -f sklearn.Dockerfile .
+```
+
+You should see an output similar to this
+
+```bash
+Sending build context to Docker daemon  110.6kB
+Step 1/6 : FROM python:3.7-slim
```

Comment:
Do we need these verbose output ? Also it may change over the time.

Reply:
so given both the sklearn and xgboost dockerfiles are in same context, we saw some red herrings when docker tries to pick up layers from the previous image built for 'kfserving' pieces. This gives clarity - again more dev oriented until these images mature and become official.